simnibs/simulation/fem.py

```diff
@@ -1463,12 +1462,22 @@ def tms_many_simulations(
             E = np.vstack([-d.dot(v) for d in D]).T * 1e3
             dAdt = dAdt[roi]
             E -= dAdt
-            if field == 'E':
-                out_field = E
-            elif field == 'J':
-                out_field = calc_J(E, cond)
-            elif field.lower() == 'vdadt':
-                out_field = (v, dAdt)
+
+            # build output fields
+            out_field = []
+            if 'E' in field:
+                out_field.append(E)
+            if 'D' in field:
+                out_field.append(dAdt)
+            if 'J' in field:
+                out_field.append(calc_J(E, cond))
+            if 'v' in field:
+                out_field.append(v)
+            out_field = tuple(out_field)
+
+            # if only one field to output, un-tuple
+            if len(out_field) == 1:
+                out_field = out_field[0]
```

Comment:
...and if only one field (the 'standard' way), untupled, to keep compatibility with old post_pro functions